modules/web/client/src/main/scala/observe/ui/components/sequence/ObsHeader.scala

```diff
@@ -42,6 +42,7 @@ object ObsHeader:
           s"${props.observation.title} [${props.observation.obsId}]"
         ),
         <.div(ObserveStyles.ObsSummaryDetails)(
+          <.span(^.id := "sequence-table-expand-all"),
```

Comment:
What is this for?

Reply:
This is the dom element the react portal attaches to from the SequenceTable